frontend/components/campaigns/campaignCharities/CampaignCharityAccordionCard.tsx

```diff
@@ -0,0 +1,67 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Box, Stack, Typography, useMediaQuery, useTheme } from '@mui/material';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import {
+  accordianSx,
+  charityDesktopDescriptionSx,
+  charityDesktopImageSx,
+  charityGraphStackSx,
+  charityLogoSx,
+  charityMobileDescriptionSx,
+  charityMobileImageSx,
+  graphSx,
+  stackSx,
+} from '../../../styles/components/charities/CampaignCharityAccordianCard';
+import { CampaignCharityDonationPublicData } from '../../../types/campaignCharities';
+import SmallCompetingGraph from '../../charts/SmallCompetingGraph';
+
+interface Props {
+  campaignCharity: CampaignCharityDonationPublicData;
+}
+
+const CampaignCharityAccordionCard = ({ campaignCharity }: Props) => {
```

Comment:
<img width="373" alt="Screenshot 2022-11-02 at 2 08 31 PM" src="https://user-images.githubusercontent.com/71375383/199411340-c360bd2e-df3c-41a0-a564-a8319084ece2.png">

Can the expanded accordion have a different background color and any other color/styling for the other elements to make it noticeable that it is expanded?

Reply:
It's quite obvious when an accordian is expanded. No need a different color to let them know.